src/components/AddBook.js

```diff
@@ -17,8 +17,7 @@ const AddNewBook = () => {
     // eslint-disable-next-line camelcase
     const item_id = Math.random();
     const category = 'Crime';
-    const author = 'A.k Khan';
-    /*     dispatch(addBook(id, title, author)); */
+    const author = 'A.K Khan';
     dispatch(addBooksAsync({
       // eslint-disable-next-line camelcase
```

Comment:
- [ ] Kindly remove the commented code on line `3`, and any other commented code for cleaner code and better readability